backend/core/settings.py

```
@@ -72,6 +72,14 @@
     }
 }
 
+# 测试环境使用内存 SQLite，避免影响开发数据库
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)